tutormfe/templates/mfe/apps/mfe/webpack.dev-tutor.config.js

```diff
@@ -1,6 +1,11 @@
 const { merge } = require('webpack-merge');
+const fs = require('fs');
 
-const baseDevConfig = require('@edx/frontend-build/config/webpack.dev.config.js');
+const baseDevConfig = (
+  !fs.existsSync('./webpack.dev.config.js')
```

Comment:
Nit: wouldn't an affirmation make the code a bit more readable, here?  (As opposed to a negation.)

Reply:
From a performance viewpoint, most MFEs use the resolved frontend-platform config file (whereas the negation)
But I'd rather have a clearer code reading instead, So I'll switch to the affirmative.